service/docs/source/geopm_pio_platformcharacterization.7.rst

```diff
@@ -0,0 +1,98 @@
+geopm_pio_platformcharacterization(7) -- IOGroup providing signals and controls for Platform Characterization
+=============================================================================================================
+
+Description
+-----------
+
+
+
+The PlatformCharacterization IOGroup implements the :doc:`geopm::IOGroup(3) <GEOPM_CXX_MAN_IOGroup.3>`
+interface to provide signals and controls used to provide per-platform characterization information.
+
+This IOGroup provides signals that describe the energy efficient frequencies for Core,
+Uncore, and GPU domains on a per node basis. This information is stored in a persistent
+cached file per node, and is populated through writing the related IOGroup controls.
+
+Requirements
+^^^^^^^^^^^^
+
+No special library support is required to use the GEOPOM PlatformCharacterization signals and controls,
+however the signals will be un-initialized by default.  System administrators, researchers, or users
+are expected to use the GEOPM experiment infrastructure (or other frameworks) to characterize nodes in
+the system.  This information may then be written to the cache file through using the IOGroup controls.
+Subsequent signal reads will then provide the values written as part of characterization for use by GEOPM
+agents and users.
+
+For basic system characterization the integrated arithimetic intensity benchmark (AIB) workload is
+recommended.
+
+The CPU Compute Activity policy recommendation script may be used to provide basic
+recommendations for CPU domain characterization.
+
+The GPU Compute Activity policy recommendation script may be used to provide basic
+recommendations for GPU domain characterization.
+
+Signals
+-------
+
+``NODE_CHARACTERIZATION::GPU_CORE_FREQUENCY_EFFICIENT``
+    GPU Compute Domain energy efficient frequency in hertz.
+
+    *  **Aggregation**: average
+    *  **Domain**: board
+    *  **Format**: double
+    *  **Unit**: hertz
+
+``NODE_CHARACTERIZATION::CPU_CORE_FREQUENCY_EFFICIENT``
```

Comment:
```suggestion
``NODE_CHARACTERIZATION::CPU_CORE_HIGH_INTENSITY_FREQUENCY_EFFICIENT``
```
The efficient frequency is (on most hardware) impacted by the hardware characteristics and the workload intensity.  As such it's worth considering the naming scheme to differentiate workload intensity.  A simple high, medium, low provides a reasonable start and fits well with the current mechanism for determining Fe (core high medium, low corresponds to something like AIB intensity 16, 4, 1 and the reverse for uncore).